Makefile

```diff
@@ -4,7 +4,7 @@
 CODE_DIRS ?= doc examples examples_flask pyvista tests
 # Files in top level directory
 CODE_FILES ?= *.py *.rst *.md
-CODESPELL_SKIP ?= "*.pyc,*.txt,*.gif,*.png,*.jpg,*.ply,*.vtk,*.vti,*.vtu,*.js,*.html,*.doctree,*.ttf,*.woff,*.woff2,*.eot,*.mp4,*.inv,*.pickle,*.ipynb,flycheck*,./.git/*,./.hypothesis/*,*.yml,doc/_build/*,./doc/images/*,./dist/*,*~,.hypothesis*,./doc/examples/*,*.mypy_cache/*,*cover,./tests/tinypages/_build/*,*/_autosummary/*"
+CODESPELL_SKIP ?= "*.pyc,*.txt,*.gif,*.png,*.jpg,*.ply,*.vtk,*.vti,*.vtu,*.js,*.html,*.doctree,*.ttf,*.woff,*.woff2,*.eot,*.mp4,*.inv,*.pickle,*.ipynb,flycheck*,./.git/*,./.hypothesis/*,*.yml,./doc/_build/*,./doc/images/*,./dist/*,*~,.hypothesis*,./doc/examples/*,*.mypy_cache/*,*cover,./tests/tinypages/_build/*,*/_autosummary/*,*/_build"
```

Comment:
I honestly don't know.  I find the expansion inside Makefile's mysterious :).

I'll try your suggestion.

Reply:
My understanding is that there's no expansion here, the literal string gets passed on the command line to codespell. How it parses the comma-separated globs is up to it. But since `*.vtu` seems to catch all `.vtu` files in the project, I'd expect globs to be matched against full paths. But I haven't experimented with it to prove either way.

Reply:
This was a good catch.  I think the addition of the `./` before `./doc/_build` caused my issues.  I don't even know why I had done this.  I agree that there seems to be a lot of duplicated paths already, which contributed some small amount to my mistake here.  Another PR though...

Reply:
I don't understand the distinction of why including the prefix `./` to a path causes problems.  But it is used in a few places here, so these parts may not even work as intended?

Reply:
If you're running codespell from the root directory of the repo then I'd expect `./doc` and `doc` to be the same. Hmm, though now that I wrote this out loud I would expect them to always be the same :laughing: No idea. I agree we need to clean this up, elsewhere, with some careful experimentation. Codespell is written in python so hopefully it just passes the patterns into a `glob()`, which would make reasoning about its behaviour ~easy~ not too difficult.

I mostly pointed this out because `_build`-related parts in here were changed back to back in https://github.com/pyvista/pyvista/pull/2108 and https://github.com/pyvista/pyvista/pull/2109...